cmd/server.go

```diff
@@ -29,9 +30,35 @@ const (
 	serverStopAction
 	serverShutdownAction
 	serverDeleteAction
+	serverCreateAction
 )
 
 var forceFlag bool
+var options = createOptions{}
+
+type createOptions struct {
+	mu            sync.Mutex
```

Comment:
There is no reason to use a mutex here. Remove.

Reply:
Did already figured out it causes an error